custom_components/tuya_local/devices/vta+_fan_heater

```diff
@@ -0,0 +1,78 @@
+name: Fan heater
+products:
+  - id: ebcb3fda096d331d27h8yc
+    name: VTA+ AXIAL Fan Heater
+primary_entity:
+  entity: climate
+  translation_only_key: heater
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: preset_mode
+          conditions:
+            - dps_val: "null"
```

Comment:
The string "null", not the null value (None in python)? This should be marked as `hidden: true` if it is just to handle an edge case, as this will shadow the level_1 preset for setting to fan_only.

Reply:
I tested `hidden: true` on the string dps_val "null", but apparently it didn't affect the feedback i get.
What happens is that when i change from `heat`  to ` fan_only` hvac_mode, the entity state becomes unknown for about 10 seconds and an error is shown, i guess while the device pushes again the state it recieved, but using the "null" dps_val, i get a state of "Fan Only - null" (which i only see in the entity but not in the card) for the same 10 seconds, but no error or unknown state is shown. As i said, no change in this behavior occurred when marking it as ` hidden: true` , so i will change it as i think this would make the code more cohesive with other devices.

Reply:
I still think this should be null without quotes if it is to cover a period when the device is not sending anything, however once it has a value, the value will not become null. so this could be a case of "tried this and it seemed to work" when in fact it is a placebo. Perhaps debug logs are needed to confirm what the device is actually sending, and to explain why this makes a difference.

Also one of the heat options should be marked hidden to get predictable behaviour.